test/usecases/CMakeLists.txt

```diff
@@ -6,7 +6,8 @@ set(NMODL_USECASE_DIRS
     parameter
     func_proc
     func_proc_pnt
-    nonspecific_current)
+    nonspecific_current
+    cabpump)
```

Comment:
the PR title/description and changes LGTM. But I am wondering if we can use/modify the existing `func_proc*` tests to cover FUNCTION/PROCEDURE related aspects. 

(i.e. I am wondering what is specific in this `cabpump` test that can't be covered by the existing test. Also thinking that the mod file looks long/complicated for its purpose?)

Reply:
I am all for simplifying it, I just took the test from https://github.com/BlueBrain/nmodl/blob/master/test/integration/mod/cabpump.mod and removed the TABLE statement. I'll see if I can reproduce it with a smaller mod file first.

Reply:
Turns out the issue occurs when the INITIAL block has a function call, and a minimal change to the `func_proc` test exposes the issue.